content/_layouts/corebaseline.html.haml

```diff
@@ -3,17 +3,20 @@ layout: developersection
 ---
 
 :ruby
-    oldest_weekly = site._generated[:core_tiers].weeklyCores[0]
+    oldest_supported_weekly = site._generated[:core_tiers].weeklyCores[0]
 
     # For this file, assume that the the entries are in increasing order and get the second column (which is the last pre-split version)
     # TODO Figure out a way to obtain this content differently, this is terrible
     split_versions = open('https://raw.githubusercontent.com/jenkinsci/jenkins/master/core/src/main/resources/jenkins/split-plugins.txt').read.lines.select { |l| l.strip.length > 0 && !(l.strip =~ /^#/) }.map { |l| l.split(' ')[1] }
     latest_split = split_versions[-1]
 
-    lts = site._generated[:core_tiers].stableCores
-    oldest_lts = lts[0]
+    supported_lts = site._generated[:core_tiers].stableCores
+    oldest_supported_lts = supported_lts[0]
+    all_lts = site._generated[:lts_releases].results.map { |v| v.version }
```

Comment:
IIUC use of this `all_lts` in `next_lts_1` and `previous_lts_highs` rather than `lts` is the fix.